pkg/api/dtos/datasource.go

```diff
@@ -28,6 +28,7 @@ type DataSource struct {
 	Version          int                    `json:"version"`
 	ReadOnly         bool                   `json:"readOnly"`
 	AccessControl    accesscontrol.Metadata `json:"accessControl,omitempty"`
+	APIVersion       string                 `json:"apiVersion"`
```

Comment:
```suggestion
    // The APIVersion when the configuration values were saved
	APIVersion       string                 `json:"apiVersion"`
```

Reply:
I am not sure if the comment is needed (and there is no other property commented here). The explanatory comments are in the SDK.This may get stale?

Reply:
maybe -- I just want to make sure we know that represents the one saved in the database, not the one conigured in plugin.json.  It will be very normal for them to be different.

As long as everyone here knows that -- then no need for comments, but I am not convinced we all understand it well yet.